Sources/SotoCore/Doc/Region.swift

```diff
@@ -68,7 +68,7 @@ public struct Region: RawRepresentable, Equatable {
     public static var mecentral1: Region { .init(rawValue: "me-central-1") }
     // Middle East (Bahrain)
     public static var mesouth1: Region { .init(rawValue: "me-south-1") }
-    // South America (São Paulo)
+    // South America (Sao Paulo)
```

Comment:
OOI - any reason for this?

Reply:
Region.swift is a generated file. That'll come from the endpoints.json file from AWS.